CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3_TEMPLATE.md

```diff
@@ -1,13 +1,13 @@
-# 1 Convert sample to use Spring Boot 3?
+# 1 Convert sample to use Spring Boot 3
 
 ## 1.1 Update your infra
 
 - [JDK17](https://www.oracle.com/java/technologies/downloads/).
 
-## 1.2 Update the pom parent
-   
-   Note: The pom parent of the samples should point to *azure-spring-boot-samples-6.x-pom.xml*.
+## 1.2 Update dependencies(optional)
 
-## 1.3 Update dependencies(optional)
+This is written in each sample's README. Most of them don't need to update.
```

Comment:
How about delete this line? This file `CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md` is used to tell customer how to convert sample to use Spring Boot 3. So, these steps should be written in `CONVERT_SAMPLE_TO_USE_SPRING_BOOT_3.md` instead of README.md.